app/src/ui/add-repository/create-repository.tsx

```diff
@@ -419,7 +419,7 @@ export class CreateRepository extends React.Component<
     return (
       <Row>
         <Select
-          label={__DARWIN__ ? 'Git Ignore' : 'Git ignore'}
+          label={__DARWIN__ ? '.gitignore' : '.gitignore'}
```

Comment:
```suggestion
          label={'.gitignore'}
```

This switch no longer would matter

Reply:
oh yeah, thanks!